docs/questions/burning-NFTs-vs-fungibles.md

```diff
@@ -0,0 +1,120 @@
+### Question
+
+
+What's the difference between burning NFTs and fungible tokens?
+
+
+### Answer
+
+
+GalaChain provides built-in mechanisms for burning both NFTs and fungible tokens through the same core types and functions, but with some key differences in how they're handled:
+
+1. Token Burn Structure:
+Both NFTs and fungible tokens use the `TokenBurn` class from `@gala-chain/api`:
+
+```typescript
+import { TokenBurn } from '@gala-chain/api';
+
+// TokenBurn class structure - used for both NFTs and fungibles
+export class TokenBurn extends ChainObject {
+  @ChainKey({ position: 0 })
+  public burnedBy: UserAlias;    // Who performed the burn
+
+  @ChainKey({ position: 1 })
+  public collection: string;      // Token collection
+
+  @ChainKey({ position: 2 })
+  public category: string;        // Token category
+
+  @ChainKey({ position: 3 })
+  public type: string;           // Token type
+
+  @ChainKey({ position: 4 })
+  public additionalKey: string;   // Additional identifier
+
+  @ChainKey({ position: 5 })
+  public instance: string;       // Token instance
+
+  public quantity: BigNumber;    // Amount burned
+}
+```
+
+2. Burn Quantity Specification:
+When burning tokens, you specify the burn details using `BurnTokenQuantity`:
+
+```typescript
+import { BurnTokenQuantity, TokenInstanceKey } from '@gala-chain/api';
+
+// For NFTs - quantity is always 1
+const nftBurn: BurnTokenQuantity = {
+  tokenInstanceKey: new TokenInstanceKey({
+    collection: 'my-nfts',
+    category: 'characters',
+    type: 'hero',
+    additionalKey: '',
+    instance: '123'  // Specific NFT instance
+  }),
+  quantity: new BigNumber(1)
+};
+
+// For fungible tokens - quantity can be any positive number
+const fungibleBurn: BurnTokenQuantity = {
+  tokenInstanceKey: new TokenInstanceKey({
+    collection: 'game-currency',
+    category: 'coins',
+    type: 'gold',
+    additionalKey: '',
+    instance: '0'  // Usually '0' for fungibles
+  }),
```

Comment:
```suggestion
  tokenInstanceKey: TokenInstanceKey.fungibleKey({
    collection: 'game-currency',
    category: 'coins',
    type: 'gold',
    additionalKey: ''
  }),
```

also a comment `Usually '0' for fungibles` is not accurate, because it's strict - `0` means it is for fungibles